test/com/facebook/buck/jvm/kotlin/KotlinTestAssumptions.java

```diff
@@ -41,4 +44,12 @@ public static void assumeCompilerAvailable(@Nullable BuckConfig config) throws I
     }
     assumeNoException(exception);
   }
+
+  /**
+   * Kotlinc doesn't support passing java files as arguments until 1.1.3.
+   * https://youtrack.jetbrains.com/issue/KT-17697
+   */
+  public static void assumeCompilerSupportsMixedSources() {
```

Comment:
Can we just use junit's `@Ignore` annotation and link it with a github issue for bumping the compiler? 
Something like `@Ingore("https....github/issues/1234")`